analysis/reachability/reachable_functions.go

```diff
@@ -159,16 +158,17 @@ func findCallees(program *ssa.Program, f *ssa.Function, action func(*ssa.Functio
 // graph to record cross-package function calls.
 //
 // The return value is a map from reachable *ssa.Function values to true.
-func FindReachable(program *ssa.Program, excludeMain bool, excludeInit bool, graph DependencyGraph) map[*ssa.Function]bool {
+func FindReachable(state *dataflow.AnalyzerState, excludeMain bool, excludeInit bool, graph DependencyGraph) map[*ssa.Function]bool {
 
-	allFunctions := ssautil.AllFunctions(program)
-	fmt.Fprintf(os.Stderr, "allFunctions contains %d total\n", len(allFunctions))
+	allFunctions := ssautil.AllFunctions(state.Program)
+	state.Logger.Infof("allFunctions contains %d total\n", len(allFunctions))
```

Comment:
nit: it might make more sense to log something more along the lines of "%d SSA functions found" since the user may not know what `allFunctions` means

Reply:
Yes, good call, those were probably more for debugging.